relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -314,15 +316,49 @@ pub fn extract_tags(
                 } else {
                     scrubbed
                 };
-                Url::parse(url).ok().and_then(|url| {
+                if let Some(domain) = Url::parse(url).ok().and_then(|url| {
                     url.domain().map(|d| {
                         let mut domain = d.to_lowercase();
                         if let Some(port) = url.port() {
                             domain = format!("{domain}:{port}");
                         }
                         domain
                     })
-                })
+                }) {
+                    Some(domain)
+                } else if let Some(server_host) = span
+                    .data
+                    .value()
+                    .and_then(|data| data.get("server.address"))
+                    .and_then(|value| value.as_str())
+                {
+                    let lowercase_host = server_host.to_lowercase();
+                    let (domain, port) = match lowercase_host.split_once(':') {
+                        Some((domain, port_str)) => {
+                            if let Ok(port) = port_str.parse::<u16>() {
+                                (domain, Some(port))
+                            } else {
+                                (domain, None)
+                            }
+                        }
+                        None => (server_host, None),
+                    };
+
+                    if let Some(url_scheme) = span
+                        .data
+                        .value()
+                        .and_then(|data| data.get("url.scheme"))
+                        .and_then(|value| value.as_str())
+                    {
+                        span_tags.insert(
+                            SpanTagKey::RawDomain,
+                            format!("{}://{}", url_scheme, server_host),
```

Comment:
I assume you'd want to use `lowercase_host` to be consistent:

```suggestion
                            format!("{url_scheme}://{lowercase_host}"),
```

Reply:
👍

Reply:
@DominikB2014 please also change the format string: `format!("{url_scheme}://{lowercase_host}")`

Reply:
done! As i'm curious, is this more for consistency sake, or is there some benefit to this?

Reply:
It's shorter and more readable, especially for format strings with many arguments. Since this is a relatively new Rust feature you will find many places in Relay which haven't been updated yet.

There is also a [clippy lint](https://rust-lang.github.io/rust-clippy/master/#/uninlined_format_args), though it is not enabled by default.

Reply:
Makes sense, thank you!